src/main/java/com/manardenza/TestUtils.java

```diff
@@ -20,10 +21,52 @@
     public static final Date RESERVED_TO = new Date(117, 1, 15);
     public static final User USER = new User(FIRST_NAME, LAST_NAME);
     public static final Room ROOM = new Room("Room", 2, 300);
-    public static final List<Room> ROOMS_LIST = new ArrayList<>();
-    public static final Hotel HOTEL = new Hotel(HOTEL_NAME, CITY, ROOMS_LIST);
+    public static final Hotel HOTEL = new Hotel(HOTEL_NAME, CITY, getRoomsSeedList());
     public static final Map<String, List<Room>> ROOMS_MAP = new HashMap<>();
 
-    private TestUtils() {}
+    public static List<Room> getRoomsSeedList() {
+        List<Room> testRoomsList = new ArrayList<>();
+        testRoomsList.add(new Room("DOUBLE ROOM", 2, 300));
+        testRoomsList.add(new Room("DOUBLE ROOM", 2, 300));
+        testRoomsList.add(new Room("DOUBLE ROOM", 2, 300));
+        testRoomsList.add(new Room("DOUBLE ROOM", 2, 300));
+        testRoomsList.add(new Room("DOUBLE ROOM", 2, 300));
+        testRoomsList.add(new Room("DOUBLE ROOM", 2, 300));
+        testRoomsList.add(new Room("DOUBLE ROOM", 2, 300));
 
+        return testRoomsList;
+    }
+
+    public static List<Reservation> getTestReservationList() {
+        List<Reservation> testReservationList = new ArrayList<>();
+        testReservationList.add(new Reservation(RESERVED_FROM, RESERVED_TO, USER, getRoomsSeedList().get(0), HOTEL));
+        testReservationList.add(new Reservation(RESERVED_FROM, RESERVED_TO, USER, getRoomsSeedList().get(1), HOTEL));
+        testReservationList.add(new Reservation(RESERVED_FROM, RESERVED_TO, USER, getRoomsSeedList().get(2), HOTEL));
+        testReservationList.add(new Reservation(RESERVED_FROM, RESERVED_TO, USER, getRoomsSeedList().get(3), HOTEL));
+
+        return testReservationList;
+    }
+
+    public static List<Room> getRoomsListToReturn() {
+        List<Room> roomsListToReturm = new ArrayList<>();
+        roomsListToReturm.add(getRoomsSeedList().get(4));
+        roomsListToReturm.add(getRoomsSeedList().get(5));
+        roomsListToReturm.add(getRoomsSeedList().get(5));
+        return roomsListToReturm;
+    }
+
+    public static Map<String, List<Room>> getTestRoomsMap() {
+        Map<String, List<Room>> testRoomsMap = new HashMap<>();
+        testRoomsMap.put(HOTEL_NAME, getRoomsSeedList());
+        return testRoomsMap;
+    }
+
+    public static Map<String, List<Room>> getTestRoomsToReturnMap() {
+        Map<String, List<Room>> testRoomsMap = new HashMap<>();
+        testRoomsMap.put(HOTEL_NAME, getRoomsListToReturn());
+        return testRoomsMap;
+    }
+
+    private TestUtils() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this constructor to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1213)